packages/@vue/cli-shared-utils/index.js

```diff
@@ -4,7 +4,8 @@
   'spinner',
   'validate',
   'openBrowser',
-  'pluginResolution'
+  'pluginResolution',
+  'babelHelpers'
```

Comment:
I don't see why this needs to be placed in `cli-shared-utils`. It's used only by `cli-plugin-babel` and should be placed in that package instead.

Reply:
because `cli-plugin-typescript` also use babel-loader, may be in the future this will need there or some where else